.prettierrc.js

```diff
@@ -1,8 +1,9 @@
 module.exports = {
   bracketSpacing: true,
-  jsxBracketSameLine: true,
+  jsxBracketSameLine: false,
   singleQuote: true,
   trailingComma: 'es5',
-  printWidth: 160,
-  tabWidth: 4
+  tabWidth: 4,
+  semi: true,
+  printWidth: 80,
```

Comment:
Should we use the `printWidth: 140` instead? (as it is pretty much the standard across all the projects)

Reply:
I've followed recommendation, but I can change this value to `140` for the consistency.
https://prettier.io/docs/en/options.html#print-width

Reply:
Yes, please change it to `140`.